linetools/spectra/io.py

```diff
@@ -124,7 +125,7 @@ def readspec(specfil, inflg=None, efil=None, verbose=False, multi_ivar=False,
             if debug:
                 print(
   'linetools.spectra.io.readspec(): Assuming separate flux and err files.')
-            xspec1d = parse_linetools_spectrum_format(hdulist)
+            xspec1d = parse_linetools_spectrum_format(hdulist, **kwargs)
```

Comment:
Mention in the docstring of readspec that kwargs are passed down to XSpectrum1D initialization.